src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Rows.component.tsx

```diff
@@ -1,27 +1,33 @@
 /** @jsx jsx */
-import { jsx, SerializedStyles } from "@emotion/core";
+import { css, jsx, SerializedStyles } from "@emotion/core";
 import TableCell from "@material-ui/core/TableCell";
 
-function chooseAlignment(element: string): "left" | "right" {
+function checkIfElementIsName(element: string): ["left" | "right", boolean] {

```

Comment:
reject: 
1. return type should be `{ alignment: "left" | "right", isName: boolean }`
2. this is to complicated for what it does and you use it only once, i would just remove this function and use the checks directly
```
const isName = element === "name"
const alignment = isName ? "left" : "right"
```

Reply:
It is no longer necessary, because of the changes in the ticket "Add all main filters". It will be changed by solving the merge conflict.